src/bin/ion/commands/beta/generate/templates/java/scalar.templ

```diff
@@ -0,0 +1,69 @@
+package {{ namespace }};
```

Comment:
(🗺️ PR tour) Contents of files `java/scalar.templ`, `java/sequence.templ`, `rust/scalar.templ`, `rust/sequence.templ` are copied from `java/class.templ` and `rust/struct.templ` respectively.